seb_openedx/middleware.py

```diff
@@ -12,9 +12,11 @@
 from seb_openedx.permissions import get_enabled_permission_classes
 from seb_openedx.seb_courseware_index import SebCoursewareIndex
 from seb_openedx.edxapp_wrapper.get_chapter_from_location import get_chapter_from_location
+from seb_openedx.user_banning import is_user_banned, ban_user
 
 SEB_WHITELIST_PATHS = getattr(settings, 'SEB_WHITELIST_PATHS', [])
 SEB_BLACKLIST_CHAPTERS = getattr(settings, 'SEB_BLACKLIST_CHAPTERS', [])
+BANNING_ENABLED = getattr(settings, 'SEB_USER_BANNING_ENABLED', False)
```

Comment:
Why is this defaulting to False, I would asume this should be always on

Reply:
Ok